helm-charts/common/data-prep/ci-videoqna-values.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+vdms-vector-db:
+  enabled: true
+
+image:
+  repository: opea/dataprep-multimodal-vdms
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+indexName: "mega-videoqna"
+vdmsHost: ""
+vdmsPort: "8001"
+entryCommand: ["/bin/sh"]
+extraArgs: ["-c", "sleep 15 && python ingest_videos.py"]
```

Comment:
I know this is inherited from docker_compose file, but this "sleep 15" looks like a workaround somewhere which should be fixed either in the Dockerfile or the python code.
Could you help file an issue or check with the owner of VideoQnA, so we can leave a comment here and to fix it later.

Reply:
This is an application level issue. I am not aware of the application owner.